src/webrtc/call.ts

```diff
@@ -1286,7 +1286,7 @@ export class MatrixCall extends EventEmitter {
         // https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Perfect_negotiation
         const offerCollision = (
             (description.type === 'offer') &&
-            (this.makingOffer || this.peerConn.signalingState != 'stable')
+            (this.makingOffer || this.peerConn?.signalingState != 'stable')
```

Comment:
I might be missing some context on the errors, but if we're getting negotiate events without a call ongoing this feels like something we should be catching at the top of the function. It's entirely possible, if they race, and doesn't necessarily indicate an error, but rolling it into the state of whether an offer collided or not feels like mixing semantics up.

Reply:
I'll try to figure out what's going on here, but that was the easiest solution if we were to merge HMR.